hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestStorageContainerManager.java

```diff
@@ -193,7 +194,7 @@ private void testRpcPermissionWithConf(
 
       try {
         ContainerWithPipeline container3 = mockClientServer
-            .allocateContainer(xceiverClientManager.getType(),
+            .allocateContainer(SCMTestUtils.getReplicationType(ozoneConf),
             HddsProtos.ReplicationFactor.ONE, "OZONE");
```

Comment:
"OZONE" could be replaced by OzoneConsts.OZONE if it's not required to be in capital letters.
Otherwise, we can add a new constant in OzoneConsts.

Reply:
This magic constant is used in multiple tests, I would suggest to resolve this in a separate JIRA after this one get merged, I have created https://issues.apache.org/jira/browse/HDDS-2378 to track this, and I am open to post a patch for that as well.